tango_simlib/utilities/fandango_json_parser.py

```diff
@@ -82,22 +82,50 @@ def preprocess_attribute_types(self, attribute_data):
             for attr_prop, attr_prop_value in attr_config.items():
                 if attr_prop == 'data_type':
                     attr_config[attr_prop] = getattr(CmdArgType, attr_prop_value)
+                    # ensuring READ_WRITE is assigned to 'writable' key for all DevEnum
```

Comment:
Why is it necessary to force DevEnums to 'writable'. I can see cases where you may want to enumerate a state (e.g. is a device a SIM device or a REAL device) but not want it to be writable. Is there something in the TANGO implementation (or ours) that requires this?

Reply:
@lvdheever According to @samtwum the file exported using Fandango doesn't accurately report whether enum attributes are read only, write only, or both.  For this reason, the most general option was selected.  It is something that should be fixed in the Fandango exporter.

Reply:
@lvdheever Just an addition to what @ajoubertza has explained...
Some of the DevEnum type attributes have 'WT_UNKNOWN' as the value registered for the attribute property 'writable' after exporting the data from the running device in the VM.
So the parser forces all values of the 'writable' attribute property to 'READ_WRITE'.